### PR TITLE
Disable unityscript and boo

### PR DESCRIPTION
**Pull request by UnityAlex**
We currently still include boo and unityscript dlls with the mono distribution. These have been deprecated and removed from usage on the Unity side for several releases. We can probably safely remove them from the mono distribution now.

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla.cds.internal.unity3d.com/Unity-Technologies/mono?pullRequest=1253) <br/>All committers have signed the CLA.

**Review by joncham — APPROVED** [review]

**Comment by UnityAlex** [comment]
This PR removes the scripts for the `build_win_bare_minimum` pipeline as all they did was build unityscript and boo. Since we are no longer bundling those with the mono builds.7z we no longer need this pipeline. I'll need someone with admin privileges to remove that PR requirement in order for this to land.

File: .yamato/build_win_bare_minimum.yml [hunk 1]
```diff
@@ -6,9 +6,4 @@ agent:
   flavor: b1.xlarge
 
 commands:
-  - .yamato/build_win_bare_minimum.bat
-
-artifacts: 
-  bareminimum:
-    paths:
-      - incomingbuilds\bareminimum\**
+  - echo YAY I DO NOTHING NOW!
```
Comment (ashwinimurt) [inline .yamato/build_win_bare_minimum.yml]: lol, let me see if I can mess with settings

Comment (ashwinimurt) [reply .yamato/build_win_bare_minimum.yml]: I removed bare_minimum from branch status check, lets undo hack and merge

**Review by ashwinimurt — APPROVED** [review]

**Pull request merged**